ecosystem/platform/server/app/views/leaderboard/it3.html.erb

```diff
@@ -102,12 +102,12 @@
 
           <%= t.with_body do %>
             <% @metrics.each_with_index do |metric, i| %>
-              <%= render TableRowComponent.new do |tr| %>
+              <%= render TableRowComponent.new(class: 'cursor-pointer select-none active:translate-y-0.5', data: { controller: 'table_row', action: 'click->table_row#tableRowClick' }) do |tr| %>
```

Comment:
I think we can set the controller and action inside the TableRowComponent instead of setting it here. This way it works anywhere.